zerver/context_processors.py

```diff
@@ -35,31 +35,32 @@ def add_settings(request):
         realm_uri = settings.SERVER_URI
 
     return {
-        'custom_logo_url':           settings.CUSTOM_LOGO_URL,
-        'register_link_disabled':    settings.REGISTER_LINK_DISABLED,
-        'login_link_disabled':       settings.LOGIN_LINK_DISABLED,
-        'about_link_disabled':       settings.ABOUT_LINK_DISABLED,
-        'show_oss_announcement':     settings.SHOW_OSS_ANNOUNCEMENT,
-        'zulip_admin':               settings.ZULIP_ADMINISTRATOR,
-        'terms_of_service':          settings.TERMS_OF_SERVICE,
-        'login_url':                 settings.HOME_NOT_LOGGED_IN,
-        'only_sso':                  settings.ONLY_SSO,
-        'external_api_path':         settings.EXTERNAL_API_PATH,
-        'external_api_uri':          settings.EXTERNAL_API_URI,
-        'external_host':             settings.EXTERNAL_HOST,
-        'external_uri_scheme':       settings.EXTERNAL_URI_SCHEME,
-        'realm_uri':                 realm_uri,
-        'server_uri':                settings.SERVER_URI,
-        'api_site_required':         settings.EXTERNAL_API_PATH != "api.zulip.com",
-        'email_integration_enabled': settings.EMAIL_GATEWAY_BOT != "",
-        'email_gateway_example':     settings.EMAIL_GATEWAY_EXAMPLE,
-        'open_realm_creation':       settings.OPEN_REALM_CREATION,
-        'password_auth_enabled':     password_auth_enabled(realm),
-        'dev_auth_enabled':          dev_auth_enabled(realm),
-        'google_auth_enabled':       google_auth_enabled(realm),
-        'github_auth_enabled':       github_auth_enabled(realm),
-        'development_environment':   settings.DEVELOPMENT,
-        'support_email':             settings.ZULIP_ADMINISTRATOR,
+        'custom_logo_url':            settings.CUSTOM_LOGO_URL,
+        'register_link_disabled':     settings.REGISTER_LINK_DISABLED,
+        'login_link_disabled':        settings.LOGIN_LINK_DISABLED,
+        'about_link_disabled':        settings.ABOUT_LINK_DISABLED,
+        'show_oss_announcement':      settings.SHOW_OSS_ANNOUNCEMENT,
+        'zulip_admin':                settings.ZULIP_ADMINISTRATOR,
+        'terms_of_service':           settings.TERMS_OF_SERVICE,
+        'login_url':                  settings.HOME_NOT_LOGGED_IN,
+        'only_sso':                   settings.ONLY_SSO,
+        'external_api_path':          settings.EXTERNAL_API_PATH,
+        'external_api_uri':           settings.EXTERNAL_API_URI,
+        'external_host':              settings.EXTERNAL_HOST,
+        'external_uri_scheme':        settings.EXTERNAL_URI_SCHEME,
```

Comment:
Also may be unnecessary; we probably want to call the feature `find_teams_link_disabled` anyway, which is short enough to not require the reindent.